images.yaml

```diff
@@ -586,11 +586,11 @@
 - name: quay.io/calico/kube-controllers
   overrideRepoName: calico-crd-installer
   patterns:
-  - pattern: '= v3.16.9'
+  - pattern: '>= v3.15.0'
     customImages:
     - dockerfileOptions:
       - |
-        FROM quay.io/giantswarm/crd-installer:0.1.0 AS installer
+        FROM quay.io/giantswarm/crd-installer:0.1.1 AS installer
```

Comment:
Pulling in some fixes and improvements from the new release https://github.com/giantswarm/crd-installer/blob/master/CHANGELOG.md#011---2021-04-28